app/models/test_track/remote/split_detail.rb

```diff
@@ -0,0 +1,30 @@
+class TestTrack::Remote::SplitDetail
+  include TestTrack::RemoteModel
+
+  collection_path '/api/v1/split_details'
+
+  attributes :name
+
+  def self.from_name(name)
+    raise "must provide a name" unless name.present?
```

Comment:
is this necessary? if you pass a blank value you'll get a 404, and if you leave the argument out you'll get a ruby error